### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by shabnam-ght**
finished my homework js1 week1

### CODE REVIEW

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Good job.
If the variable is never reassigned. Use `const` instead of `let`, please.

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Perfect job shabnam.
Just one notice. If the variable is never reassigned. Use `const` instead of `let`, please.

File: javascript/javascript1/week1/Goodboy-Oldboy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const dogYearOfBirth = 2015;
+let dogYearFuture = 2045;
+let dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+let inHumanYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
```
Comment (zuhairtaha) [inline javascript/javascript1/week1/Goodboy-Oldboy.js:5]: These variables are never reassigned. Use `const` instead

File: javascript/javascript1/week1/Ez-Namey.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const firstWords = ["big", "beautiful", "amazing", "small", "nice", "ugly", "delicious", "red", "awsome", "lovely"];
+
+const secondWords = ["corprate", "institute", "company", "shop", "market", "application", "website", "restaurant ", "coffeeshop", "business"];
+
+let randomNumber = Math.floor(Math.random() * 10);
+let randomNumber2 = Math.floor(Math.random() * 10);
+let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber2];
```
Comment (zuhairtaha) [inline javascript/javascript1/week1/Ez-Namey.js:7]: These variables are never reassigned. Use `const` instead

File: javascript/javascript1/week1/Age-ify.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1999;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
```
Comment (zuhairtaha) [inline javascript/javascript1/week1/Age-ify.js:3]: These variables are never reassigned. Use `const` instead

**Review by zuhairtaha — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week1/Age-ify.js (hunk 3 above)
Comment (shabnam-ght) [reply javascript/javascript1/week1/Age-ify.js]: ready to merge now?

**Pull request merged**